### ISSUE TITLE
Nexcloud / swag / duckdns : landing on SWAG index page ?

### ISSUE THREAD

**Issue by rtibby** [linuxserver/docker-swag#172]
Hello.
Trying to install Nextcloud on OMV5 (MariaDV, swag ) .
Done everything in the guide : https://forum.openmediavault.org/index.php?thread/28216-how-to-nextcloud-with-swag-letsencrypt-using-omv-and-docker-compose/

When opening in a tab the url for Nextcloud (local IP or duckdns one) , it is landing on the swag parking page  index.html instance instead of  Nexcloud one.

WHY?

![image](https://user-images.githubusercontent.com/73258472/138567993-4b62ae5f-7f5c-4e81-bb9f-005528134150.png)

**Comment by drizuid** [comment]
note, we do not offer support or validation of external guides. You must fill out the template fully to get support. as the reply to you stated, you simply did this wrong. closing this due to lack of activity.

**Comment by arkharim** [comment]
**Expected Behavior**

Connect to nextcloud.

**Current Behavior**

Connecto to Welcome swag screen
![imagen](https://user-images.githubusercontent.com/72758421/150579834-a49257cf-40b7-418b-8830-032392a91032.png)


**Steps to Reproduce**
Follow the guide: https://docs.linuxserver.io/general/swag#nextcloud-subdomain-reverse-proxy-example

In a empty docker file copy:

version: "2.1"
services:
  nextcloud:
    image: ghcr.io/linuxserver/nextcloud
    container_name: nextcloud
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/London
    volumes:
      - /home/aptalca/appdata/nextcloud/config:/config
      - /home/aptalca/appdata/nextcloud/data:/data
    depends_on:
      - mariadb
    restart: unless-stopped
  mariadb:
    image: ghcr.io/linuxserver/mariadb
    container_name: mariadb
    environment:
      - PUID=1000
      - PGID=1000
      - MYSQL_ROOT_PASSWORD=mariadbpassword
      - TZ=Europe/London
      - MYSQL_DATABASE=nextcloud
      - MYSQL_USER=ncuser
      - MYSQL_PASSWORD=ncpassword
    volumes:
      - /home/aptalca/appdata/mariadb:/config
    restart: unless-stopped
  swag:
    image: ghcr.io/linuxserver/swag
    container_name: swag
    cap_add:
      - NET_ADMIN
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/London
      - URL=linuxserver-test.duckdns.org
      - SUBDOMAINS=wildcard
      - VALIDATION=duckdns
      - DUCKDNSTOKEN=97654867496t0877648659765854
    volumes:
      - /home/aptalca/appdata/swag:/config
    ports:
      - 443:443
      - 80:80
    restart: unless-stopped

And change TZ to my location and update url and duckdnstoken. Already tested with other images.

2- Rename: 
![imagen](https://user-images.githubusercontent.com/72758421/150581934-70f6e3cc-fd17-45ff-a94f-75b821a0454b.png)


**Environment**

OS: Raspbian CPU architecture: arm32 How docker service was installed:  _curl -sSL https://get.docker.com | sh_ ; s_udo usermod -aG docker user_ ; install docker compose dependencies and _pip3 install docker-compose_

**Command used to create docker container (run/create/compose/screenshot)**
![imagen](https://user-images.githubusercontent.com/72758421/150581239-49b2715f-6df6-42b8-971a-83a53647ce3a.png)

**Docker logs**

docker logs swag -f:
![imagen](https://user-images.githubusercontent.com/72758421/150581612-9fa0f791-593b-44ce-9434-6caaa2e5ed65.png)

Can you help me? 

Thanks in advance.

**Comment by aptalca** [comment]
How did you rename the Nextcloud proxy conf? Post output of `ls -al /home/aptalca/appdata/swag/nginx/proxy-confs/*.conf`

**Comment by arkharim** [comment]
Thanks for your help! I see my mistake. I was following the guide with initial examples but in the nextcloud example the config location is in other route, my mistake to not check more carefully the configuration. I was renaming other proxy-confs/* 

On the other hand, it seems that the */nginx/proxy-confs was not created when I copy the nextcloud example and change the volume location. It was fixed removing de image and downloading again. 

Finally, with the output of ls -al /home/aptalca/appdata/swag/nginx/proxy-confs/*.conf:
![imagen](https://user-images.githubusercontent.com/72758421/150631826-6b227174-3399-42a0-a8ba-516d61f70262.png)

I swag was not starting due to: nginx: [emerg] "proxy_max_temp_file_size" directive invalid value in /config/nginx/proxy-confs/nextcloud.subdomain.conf:34

 The output of docker logs swag -f :
![imagen](https://user-images.githubusercontent.com/72758421/150631907-70c45e04-04ec-49f7-9629-bba0e5d8f80e.png)

I don't remember where I saw but this error is fixed changing 2048 to 1024
![imagen](https://user-images.githubusercontent.com/72758421/150631996-f70dd8e7-0168-48ed-942d-2c225e79b645.png)
